tests/IVIMmodels/unit_tests/generic_0.json

```diff
@@ -0,0 +1,251 @@
+            0.3666359334632083
+        ]
+    },
+    "Left kidney cortex": {
+        "noise": 0.0005,
+        "D": 0.00212,
+        "f": 0.09699999999999999,
+        "Dp": 0.019999999999999997,
+        "data": [
+            0.25738176819318737,
+            0.2930962800836272,
+            0.2955176506205483
+        ]
+    },
+    "left kidney medulla": {
+        "noise": 0.0005,
+        "D": 0.0020900000000000003,
+        "f": 0.158,
+        "Dp": 0.018999999999999996,
+        "data": [
+            0.3108687796446044,
+            0.36570256580418625,
+            0.36670702791220644
+        ]
+    },
+    "spleen": {
+        "noise": 0.0005,
+        "D": 0.0012999999999999997,
+        "f": 0.19999999999999998,
+        "Dp": 0.03000000000000001,
+        "data": [
+            0.21808329544936134,
+            0.2632012699287378,
+            0.26517466842411536
+        ]
+    },
+    "Artery": {
+        "noise": 0.0005,
+        "D": 0.0029999999999999988,
+        "f": 1.0,
+        "Dp": 0.09999999999999998,
+        "data": [
+            0.0077350704782766415,
+            0.4253636332664325,
+            0.47059116393638195
+        ]
+    },
+    "Vein": {
+        "noise": 0.0005,
+        "D": 0.002999999999999999,
+        "f": 1.0,
+        "Dp": 0.09999999999999999,
+        "data": [
+            0.004953076946444624,
+            0.2195818927759774,
+            0.24339538710853775
+        ]
+    },
+    "asc lower intestine": {
+        "noise": 0.0005,
+        "D": 0.00131,
+        "f": 0.6899999999999998,
+        "Dp": 0.028999999999999998,
+        "data": [
+            0.07293223531680129,
+            0.13939703246966545,
+            0.14260229273889413
+        ]
+    },
+    "trans lower intestine": {
+        "noise": 0.0005,
+        "D": 0.0013099999999999997,
+        "f": 0.6899999999999997,
+        "Dp": 0.029000000000000012,
+        "data": [
+            0.07335412435989602,
+            0.13954086322304496,
+            0.14304008992271988
+        ]
+    },
+    "desc lower intestine": {
+        "noise": 0.0005,
+        "D": 0.00131,
+        "f": 0.6899999999999997,
+        "Dp": 0.029000000000000015,
+        "data": [
+            0.07355121174313206,
+            0.1395099378569396,
+            0.1428453482667964
+        ]
+    },
+    "small intestine": {
+        "noise": 0.0005,
+        "D": 0.00131,
+        "f": 0.6899999999999998,
+        "Dp": 0.02900000000000001,
+        "data": [
+            0.07319218181541227,
+            0.13976493813896818,
+            0.14229073561367692
+        ]
+    },
+    "pericardium": {
+        "noise": 0.0005,
+        "D": 0.0030000000000000005,
+        "f": 0.07000000000000002,
+        "Dp": 0.009999999999999998,
+        "data": [
+            0.10069801124430204,
+            0.11354496636645572,
+            0.11515572103570605
+        ]
+    },
+    "config": {
+        "bvalues": [
+            43.0,
```

Comment:
Something isn't right with these bvalues.

Reply:
Ohh, that is true, I will change it. Those values I gave while testing the script in the command line.